examples/social/tests/cross_chain.rs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Integration tests for the Fungible Token application.
+
+#![cfg(not(target_arch = "wasm32"))]
+
+//use async_graphql::InputType;
+use linera_sdk::test::TestValidator;
+use social::Operation;
+
+/// Test posting messages across microchains.
+///
+/// Creates the application on chain2 indirectrly from the subscription, then
+/// send a message to chain2 and see it receiced on chain1.
+#[tokio::test]
+async fn cross_chain_transfer() {
+    let (validator, bytecode_id) = TestValidator::with_current_bytecode().await;
+    let mut chain1 = validator.new_chain().await;
+
+    // Initialization is trivial for the social app
+    let application_id = chain1
+        .create_application(bytecode_id, vec![], vec![], vec![])
+        .await;
+
+    let chain2 = validator.new_chain().await;
+
+    // Subscribe chain1 to chain2
+
+    chain1
+        .add_block(|block| {
+            block.with_operation(application_id, Operation::RequestSubscribe(chain2.id()));
+        })
+        .await;
+
+    // Make chain2 handle that fact.
+
+    chain2.handle_received_effects().await;
+
+    // Post on chain2
+
+    chain2
+        .add_block(|block| {
+            block.with_operation(
+                application_id,
+                Operation::Post("Linera is the new Mastodon".to_string()),
+            );
+        })
+        .await;
+
+    // Now make chain1 handle that fact.
+
+    chain1.handle_received_effects().await;
+
+    // Querying the own posts
+
```

Comment:
nit: We don't need as many newlines everywhere, especially when the comment applies to what immediately follows.

Reply:
Ok.